src/KubernetesClient.csproj

```diff
@@ -1,15 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
Moving back to netcore from netstandard will make the library incompatible with .net.
We should target .net standard.

Reply:
Looks like new method CopyWithPrivateKey has not been made part of the netstandard APIs. :(

Reply:
Tagging @bartonjs who can shed some more light on this.

Reply:
CopyWithPrivateKey and the CertificateRequest API are not part of .NET Standard and do not work on .NET Framework.

Reply:
Thanks Jeremy @bartonjs. Is there any plan to add it in later releases.
@brendandburns We can then target dotnetcore and the library won't work for .net :( . 
@firazs for .net we will still need to invoke openssl to create the pfx for the client cert.

Reply:
> @bartonjs. Is there any plan to add it in later releases.

I'm not personally aware of how we're going to rev netstandard.  But once someone informs me of how it's supposed to work, I'll be arguing that this stuff belongs :smile:.  Doin' what I can, and hoping that things go fairly fast.